nncf/onnx/quantization/quantize.py

```diff
@@ -55,6 +55,10 @@ def quantize_impl(model: onnx.ModelProto,
         fast_bias_correction=fast_bias_correction
     )
 
+    from nncf.quantization.algorithms.min_max.algorithm import MinMaxQuantization
+    min_max_params = quantization_parameters.algorithms[MinMaxQuantization]
+    quantization_parameters.algorithms = {MinMaxQuantization: min_max_params}
+
```

Comment:
Checked on mobilenet_v2, metrics and speed up the same as there were before.
CC: @kshpv